### PR TITLE
Fixes #3 - Overflow when converting to display text

### LINKED ISSUE #3: Overflow when converting to display text
relates to: Open-Systems-Pharmacology/PK-Sim#165

### PR DESCRIPTION
**Pull request by rwmcintosh**
The crash was happening when converting the string representation into the formatted string version.

An event takes care of that, it reads the string from the editor, converts to the property type (int in this case) and then converts back to string with the formatter.

We needed to take care of overflow in this sequence.

### CODE REVIEW

File: tests/OSPSuite.DataBinding.DevExpress.Starter/FormTest.cs [hunk 1]
```diff
@@ -87,7 +87,7 @@ private void InitBinding()
          _screenBinder.Bind(item => item.Value).To(textEditDouble);
          _screenBinder.Bind(item => item.Log).To(memoEdit);
 
-         _screenBinder.Bind(item => item.NullableValue).To(textNullableEdit);
+         _screenBinder.Bind(item => item.IntValue).To(textNullableEdit);
```
Comment (msevestre) [inline tests/OSPSuite.DataBinding.DevExpress.Starter/FormTest.cs]: Can you add another text box instead?

**Review by msevestre — CHANGES_REQUESTED** [review]

File: tests/OSPSuite.DataBinding.DevExpress.Starter/FormTest.cs (hunk 1 above)
Comment (rwmcintosh) [reply tests/OSPSuite.DataBinding.DevExpress.Starter/FormTest.cs]: I did at first. But then I could not build anymore. All the references to Devexpress are broken.
I can continue to try and figure that out, revert the change, or leave it as-is.

Something weird when I use the designer on that form.

**Comment by msevestre** [comment]
well at the very least, the variable textNullableEdit should be renamed.
But i'd rather not remove A to fix B

**Comment by rwmcintosh** [comment]
It was because of some ambiguity in the namespace that the designer is not
meant to handle.

I added another textedit, but then had to go and hand edit the designer
file.

**Pull request merged**